app/views/media_objects/_item_view.html.erb

```diff
@@ -21,7 +21,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 
 <% @poster_url = request.base_url + @currentStreamInfo[:poster_image] unless @currentStreamInfo[:poster_image].nil? %>
 <% @page_type = @currentStreamInfo[:is_video] ? 'video' : 'audio' %>
-<% can_stream = current_user && @media_object.current_checkout(current_user.id).present? %>
+<% can_stream = lending_enabled?(@media_object) ? current_user && @media_object.current_checkout(current_user.id).present? : current_user %> 
```

Comment:
I can't recall what `current_user` is if a user isn't signed in.  Would it be safer to return `true` in the non-CDL case?
```suggestion
<% can_stream = lending_enabled?(@media_object) ? current_user && @media_object.current_checkout(current_user.id).present? : true %> 
```